experimental/GroebnerWalk/src/common.jl

```diff
@@ -0,0 +1,161 @@
+
+@doc raw"""
+    groebner_walk(
+      I::MPolyIdeal, 
+      target::MonomialOrdering = lex(base_ring(I)),
+      start::MonomialOrdering = default_ordering(base_ring(I));
+      perturbation_degree = length(gens(base_ring(I))),
+      algorithm::Symbol = :standard
+    )
+
+Compute a reduced Groebner basis w.r.t. to a monomial ordering by converting it using the Groebner Walk.
+The Groebner Walk is proposed by Collart, Kalkbrener & Mall (1997).
```

Comment:
Please put your references into `doc/oscar_references.bib` and cite them here using `[BIBKEY](@cite)`.
For more details, see https://docs.oscar-system.org/dev/DeveloperDocumentation/documentation/#Updating-the-bibliography